src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1848,6 +1848,21 @@ PUBLIC_API STATUS restartIce(PRtcPeerConnection);
  */
 PUBLIC_API STATUS closePeerConnection(PRtcPeerConnection);
 
+/**
```

Comment:
Allows for pre-fetching of the host + stun candidates

Reply:
Wondering if you can modify the sample to use the new function? (testing the code path)

Reply:
@sirknightj I'll modify the sample and check for if a test case can be added.